Mathlib/Data/ENNReal/Operations.lean

```diff
@@ -169,6 +169,10 @@ theorem not_lt_top {x : ℝ≥0∞} : ¬x < ∞ ↔ x = ∞ := by rw [lt_top_iff
 
 theorem add_ne_top : a + b ≠ ∞ ↔ a ≠ ∞ ∧ b ≠ ∞ := by simpa only [lt_top_iff_ne_top] using add_lt_top
 
+@[aesop (rule_sets := [finiteness]) safe apply]
+private theorem add_ne_top' {a b : ℝ≥0∞} (ha : a ≠ ∞) (hb : b ≠ ∞) : a + b ≠ ∞ :=
```

Comment:
It's very frustrating that marking this private means the output of `finiteness?` can't be "Try this"-d into a proof, in most cases it seems like `finiteness?` isn't nearly as helpful as one would want...

Reply:
I tend to agree with you. I've made them protected in the `Finiteness` namespace